### ISSUE TITLE
[VS Editor tooling]: no auto-generation of skeleton xml docs after typing '///<' in VS2022

### ISSUE THREAD

**Issue by kMutagene** [dotnet/fsharp#14699]
The VS 2022 integration of F# seems to have dropped auto generation of XML docs sections, as well as green warning lines related to xml sections.  It worked in VS 2019.

**Repro steps**

Provide the steps required to reproduce the problem:

With auto generation i mean the generation of the XML skeleton after starting to type `///<` above a language construct, e.g.

```fsharp
/// <summary>
/// 
/// </summary>
/// <param name="someArg"></param>
```

after typing

```fsharp
///<
let f someArg = 42
```

**Expected behavior**

- Auto generation of the XML skeleton when editing a fsharp source file.

- Green warning lines in XML docs like in VS2019, e.g. when a param description is missing:

![image](https://user-images.githubusercontent.com/21338071/216915467-c50e27cc-5859-4f3a-8e7a-a5becf8202d9.png)

**Actual behavior**

- Nothing happens after typing `///<`. 

- no warnings in xml docs:

![image](https://user-images.githubusercontent.com/21338071/216915886-52cd6a2c-0a78-4581-9dda-431541f91e5b.png)

**Known workarounds**

Use VS2019.

**Related information**

Provide any related information (optional):

* Win 11
* .NET Core 6.0 +
* Visual Studio 17.4.4 (works in VS 16.11.23)

**Comment by vzarytovskii** [comment]
Hm, interesting, that should work, yes. We'll fix it

**Comment by kMutagene** [comment]
@vzarytovskii Is this supposed to be live already? Im on 17.5.4 and it is still not working.

**Comment by vzarytovskii** [comment]
> @vzarytovskii Is this supposed to be live already? Im on 17.5.4 and it is still not working.

No, it merged to 17.7
